{"uri":"file:///home/tarcisio/dev/phpfin/vendor/symfony/debug/Tests/ErrorHandlerTest.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Symfony\\Component\\Debug\\Tests","location":[11,0,11,40]},{"kind":1,"name":"TestCase","modifiers":4096,"location":[13,4,13,30],"associated":[{"kind":1,"name":"PHPUnit\\Framework\\TestCase"}]},{"kind":1,"name":"LogLevel","modifiers":4096,"location":[14,4,14,20],"associated":[{"kind":1,"name":"Psr\\Log\\LogLevel"}]},{"kind":1,"name":"BufferingLogger","modifiers":4096,"location":[15,4,15,43],"associated":[{"kind":1,"name":"Symfony\\Component\\Debug\\BufferingLogger"}]},{"kind":1,"name":"ErrorHandler","modifiers":4096,"location":[16,4,16,40],"associated":[{"kind":1,"name":"Symfony\\Component\\Debug\\ErrorHandler"}]},{"kind":1,"name":"ContextErrorException","modifiers":4096,"location":[17,4,17,59],"associated":[{"kind":1,"name":"Symfony\\Component\\Debug\\Exception\\ContextErrorException"}]},{"kind":1,"name":"SilencedErrorContext","modifiers":4096,"location":[18,4,18,58],"associated":[{"kind":1,"name":"Symfony\\Component\\Debug\\Exception\\SilencedErrorContext"}]},{"kind":1,"name":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest","location":[26,0,530,1],"associated":[{"kind":1,"name":"PHPUnit\\Framework\\TestCase"}],"description":"ErrorHandlerTest.","children":[{"kind":32,"name":"testRegister","modifiers":1,"location":[28,4,66,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testNotice","modifiers":1,"location":[68,4,97,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"triggerNotice","modifiers":36,"location":[100,4,105,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest","children":[{"kind":128,"name":"$that","location":[100,42,100,47],"scope":"triggerNotice"}]},{"kind":32,"name":"testConstruct","modifiers":1,"location":[107,4,117,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testDefaultLogger","modifiers":1,"location":[119,4,151,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testHandleError","modifiers":1,"location":[153,4,253,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testHandleUserError","modifiers":1,"location":[255,4,275,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testHandleDeprecation","modifiers":1,"location":[277,4,297,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testHandleException","modifiers":1,"location":[299,4,338,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testErrorStacking","modifiers":1,"location":[340,4,367,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testBootstrappingLogger","modifiers":1,"location":[369,4,418,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testSettingLoggerWhenExceptionIsBuffered","modifiers":1,"location":[420,4,437,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testHandleFatalError","modifiers":1,"location":[439,4,477,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testHandleErrorException","modifiers":1,"location":[482,4,495,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"},{"kind":32,"name":"testHandleFatalErrorOnHHVM","modifiers":1,"location":[497,4,529,5],"scope":"Symfony\\Component\\Debug\\Tests\\ErrorHandlerTest"}]}]}}